### PR TITLE
Feature UI extract metadata and extract text

### PR DESCRIPTION
**Pull request by danielaSantaCruz**

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/view/Models/ExtractTextRequestForm.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Daniela Santa Cruz Andrade
+ * @version 1.0
```
Comment (JRojas999) [inline src/main/java/org/fundacion/jala/converter/view/Models/ExtractTextRequestForm.java]: We agreed not to put the version. It should be deleted

Comment (danielaSantaCruz) [reply src/main/java/org/fundacion/jala/converter/view/Models/ExtractTextRequestForm.java]: Ok, already fixed

File: src/main/java/org/fundacion/jala/converter/view/metadata/ExportingFormat.java [hunk 2]
```diff
@@ -10,16 +10,18 @@
 package org.fundacion.jala.converter.view.metadata;
 
 import org.fundacion.jala.converter.view.utilities.JLabelStyle;
-import javax.swing.JPanel;
-import javax.swing.JCheckBox;
+
+import javax.swing.*;
```
Comment (JRojas999) [inline src/main/java/org/fundacion/jala/converter/view/metadata/ExportingFormat.java]: It is necessary to specify more than *

File: src/main/java/org/fundacion/jala/converter/view/metadata/MetaDataInterface.java [hunk 3]
```diff
@@ -4,11 +4,16 @@
  * This software is the confidential and proprietary information of Fundacion Jala
  * ("Confidential Information"). You shall not disclose such Confidential
  * Information and shall use it only in accordance with the terms of the
- * license agreement you entered into with Fundacion Jala
+ * license agreement you entered into with Fundacion Jala.
```
Comment (JRojas999) [inline src/main/java/org/fundacion/jala/converter/view/metadata/MetaDataInterface.java:7]: Author is missing

File: src/main/java/org/fundacion/jala/converter/view/metadata/ExportingFormat.java (hunk 2 above)
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/view/metadata/ExportingFormat.java]: specify the swing, try not to use *

File: src/main/java/org/fundacion/jala/converter/view/metadata/MetaDataInterface.java (hunk 3 above)
Comment (danielaSantaCruz) [reply src/main/java/org/fundacion/jala/converter/view/metadata/MetaDataInterface.java:7]: I am not the author of this class 😅

**Review by Paola-Xaq — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/view/metadata/ExportingFormat.java (hunk 2 above)
Comment (danielaSantaCruz) [reply src/main/java/org/fundacion/jala/converter/view/metadata/ExportingFormat.java]: Oki, already fixed

Comment (danielaSantaCruz) [reply src/main/java/org/fundacion/jala/converter/view/metadata/ExportingFormat.java]: ok, already fixed

**Review by JRojas999 — APPROVED** [review]

**Review by GustavoZHA — APPROVED** [review]

**Pull request merged**